libraries/helpers.rb

```diff
@@ -3,23 +3,25 @@ module Cookbook
     module Helpers
       # Initalize the configuration options given the attributes
       def osl_rt_init_config
+        rt_secrets = data_bag_item(*node['osl-rt']['data-bag'])
```

Comment:
This attribute doesn't have a default setting at all in the default attributes file. Also I prefer to have the same databag name but the item can be attribute. For this I'd prefer to have the databag name be `request-tracker` and the default item name could be `default`.